samples/scripts/PowerShell/GenerateOpenApiYaml.ps1

```diff
@@ -0,0 +1,34 @@
+<#
+.SYNOPSIS 
+Generates the OpenApi doc for the specified version and compares it with the baseline to make sure no breaking changes are introduced
+.Parameter SwaggerDir
+The working directory
+.PARAMETER AssemblyDir
+Path for the web projects dll
+.PARAMETER Versions
+Api versions to generate the OpenApiDoc for and compare with baseline
+.PARAMETER SwashbuckleCLIVersion
+Version of SwashbuckleCLI to use
+#>
+
+param(
+    [string]$SwaggerDir,
+
+    [string]$AssemblyDir,
+
+    [String[]]$Versions,
+
+    [string]$SwashbuckleCLIVersion = '6.1.4'
+)
+foreach ($Version in $Versions)
+{
+    write-host "Generating Yaml file for $Version"
+
+    dotnet new tool-manifest --force
+    dotnet tool install --version $SwashbuckleCLIVersion Swashbuckle.AspNetCore.Cli
+    dotnet swagger tofile --yaml --output "$SwaggerDir/$Version.yaml" "$AssemblyDir" $Version
+
+    write-host "Running comparison with baseline for version $Version"
+    docker run --rm -t -v ${SwaggerDir}:/swagger openapitools/openapi-diff:latest@sha256:5da8291d3947414491e4c62de74f8fc1ee573a88461fb2fb09979ecb5ea5eb02 "/swagger/$Version/swagger.yaml" "/swagger/$version.yaml" --fail-on-incompatible
```

Comment:
Should we build the docker image once before all of the versions run? That was the stdout is more organized?

Reply:
Used create to have them docker image pulled before each version run